src/plugins/embeddable/public/lib/panel/_embeddable_panel.scss

```diff
@@ -106,6 +106,17 @@
   &:focus {
     background-color: $euiFocusBackgroundColor;
   }
+
+}
+
+.embPanel__optionsMenuPopover-notification::after {
+  position: absolute;
+  top: 0;
+  right: 0;
+  content: '•';
```

Comment:
@MichaelMarcialis, Adding EuiIcon anyway will require styles for correct position. For me in that case using pseudo element is good solution. Could you please describe a reason why is better adding new element instead of using pseudo element?

Reply:
The only reason I asked was for the sake of consistency with other areas of Kibana that use the `dot` icon. That consistency would really only be in regard to size in this case, since the icon is a simple circle. That said, it's not a huge deal either way. If positioning an `EuiIcon` instead of a pseudo element is more complex, I'm OK keeping as you have it.

Reply:
@MichaelMarcialis, Could you please tell me where we have the same dot-notification idea in kibana?

Reply:
I believe this is the first use of a `dot` icon to draw attention to a menu button in Kibana, but we do utilize the `dot` icon in a variety of other across Kibana as a means to visually offset or grab attention from users. Some that immediately jump to mind are 1) for indicating change/dirty state when making edits on some pages (like user profiles) and 2) for indicating incompatibility of quick functions in Lens.